simulator/Cargo.toml

```diff
@@ -24,5 +24,6 @@ circle-ci = { repository = "jamwaffles/embedded-graphics", branch = "master" }
 sdl2 = "0.32.2"
 
 [dependencies.embedded-graphics]
-version = "0.5.0"
+version = "0.5.2-alpha.0"
+path = "../embedded-graphics"
```

Comment:
Can you remove this from the PR? Testing major changes like this needs a better story, but I don't want to release the simulator referencing a local path :slightly_smiling_face:

Reply:
How can I remove this without breaking CI?

Reply:
Hm, not sure to be honest. Leave it for now and I'll fix it forward after merge/before release.